tests/test_schema.py

```diff
@@ -105,6 +106,9 @@ class Unit(dj.Part):
 
     test_schema.drop()
 
+def test_list_tables():
+    assert(['#a', '#argmax_test', '#data_a', '#data_b', '#i_j', '#j_i', '#l', '#t_test_update', '__b', '__b__c', '__d', '__e', '__e__f', 'f', 'reserved_word'] == schema_simple.list_tables())
```

Comment:
Can you also break this up based on line character limit of 95? In VSCode, I usually use a ruler to remind me throughout the code. As a tip, here is some of the values in my `settings.json` that I like using.
```json
{
    "python.linting.pycodestyleEnabled": true,
    "python.linting.pylintEnabled": true,
    "python.linting.enabled": true,
    "editor.rulers": [
        95
    ],
    "python.linting.pycodestyleArgs": ["--max-line-length=95"],
}
```